src/extensibility/node/ExtensionManagerDomain.js

```diff
@@ -90,6 +90,20 @@ function _removeFailedInstallation(installDirectory) {
 function _performInstall(packagePath, installDirectory, validationResult, callback) {
     validationResult.installedTo = installDirectory;
 
+    var fail = function (err) {
```

Comment:
question [NIT]: can you refresh my memory if we tend to have function expressions like this over function definitions?